docs/question/index.md

```diff
@@ -115,7 +127,28 @@ The different types of dependency properties available are summarized in this ta
 | `clientFilesCourseStyles`    | The styles required by this question relative to `[course directory]/clientFilesCourse`.          |
 | `clientFilesCourseScripts`   | The scripts required by this question relative to `[course directory]/clientFilesCourse`.         |
 
-## Question `question.html`
+Additional details about how to access these fields from `server.py` can be found in the [`server.py` documentation](server.md#accessing-files-on-disk).
+
+### Non-randomized questions
```

Comment:
I moved these sections up so that all the documentation about question configuration is together.